app/src/components/Organization/OrgConfigForm.tsx

```diff
@@ -0,0 +1,213 @@
+import { useWeb3React } from '@web3-react/core'
+import Spinner from 'components/Spinner/Spinner'
+import { ethers } from 'ethers'
+import { addOrganizationConfig } from 'hooks/organization/functions'
+import router from 'next/router'
+import React, { useState } from 'react'
+
+const initialData = {
+  requiredTaskApprovals: 1,
+  requiredConfirmations: 1,
+  rewardMultiplier: 1,
+  rewardToken: ethers.constants.AddressZero,
+  rewardSlashDivisor: 1,
+  slashRewardEvery: 1,
+}
+
+type OrgData = typeof initialData & { [keyof: string]: any }
+
+const OrgConfigForm: React.FC<{ orgId: number }> = ({ orgId }) => {
+  const [status, setStatus] = useState({ text: '', error: false })
+  const [orgData, setOrgData] = useState<OrgData>(initialData)
+  const [processing, setProcessing] = useState(false)
+  const { account, library } = useWeb3React()
+
+  const handleChange = (ev: any) => {
+    const targetName = ev.target.name
+    let targetValue: string | number = ev.target.value
+
+    if (ev.target.type === 'number') {
+      if (targetValue) {
+        targetValue = Number(targetValue)
+      }
+    }
+
+    if (ev.target.dataset.type === 'addressArray') {
+      if (ev.key === 'Enter' && targetValue) {
+        setOrgData((prev: any) => ({
+          ...prev,
+          [targetName]: [...prev[targetName], targetValue],
+        }))
+        ev.target.value = ''
+      }
+      return
+    }
+    setOrgData((prev: any) => ({ ...prev, [targetName]: targetValue }))
+  }
+
+  const addOrgConfig = async () => {
+    if (!account) {
+      setStatus({ text: 'Wallet Not Connected', error: true })
+      return
+    }
+    setProcessing(true)
+    try {
+      const tx = await addOrganizationConfig(
+        orgId,
+        orgData.requiredTaskApprovals,
+        orgData.requiredConfirmations,
+        orgData.rewardMultiplier,
```

Comment:
This should be a bigNumber, check [here](https://github.com/innovation-upstream/buildstream/pull/53/files#diff-9cb1e1c42a3244bcaa1747e3ca53fa2e9a0b333faa633dee5a421b952b77704aL75)